examples/nlp/lasertagger/lasertagger_main.py

```diff
@@ -0,0 +1,368 @@
+import json
```

Comment:
Please add NVIDIA's license header here, see how other examples are doing it (just copy-paste it from there)

Reply:
will do.